tests/sdk_test/sdk_test.cpp

```diff
@@ -648,16 +648,11 @@ bool MegaChatApiTest::TEST_ResumeSession(unsigned int accountIndex)
     megaChatApi[accountIndex]->addChatRequestListener(this);
     megaChatApi[accountIndex]->addChatListener(this);
     MegaChatApi::setLoggerObject(logger);
-    // back to enabled: init + fetchnodes + connect
-    ASSERT_CHAT_TEST(megaChatApi[accountIndex]->init(session) == MegaChatApi::INIT_NO_CACHE,
+    ASSERT_CHAT_TEST(megaChatApi[accountIndex]->enableChat(session),
                      "Wrong chat initialization state. Expected: " + std::to_string(MegaChatApi::INIT_NO_CACHE) + "   Received: " + std::to_string(megaChatApi[accountIndex]->init(session)));
```

Comment:
Adjust the messages in case of assertion failed to be consistent with the new call to `enableChat()`